app/build.gradle

```diff
@@ -216,6 +216,7 @@ dependencies {
     androidTestImplementation "org.jetbrains.kotlin:kotlin-reflect:${kotlin_version}"
     compileOnly files('libs/amazon-device-messaging-1.0.1.jar')
     testImplementation files('libs/amazon-device-messaging-1.0.1.jar')
+    testImplementation "org.mozilla.appservices:full-megazord-forUnitTests:0.37.1"
```

Comment:
Where did this version come from? Can you add a link so folks know where to find the latest version?

Also, what happens if this dependency gets out of sync with the dependency automatically provided by a-c? Can you follow-up with the app-services team?

Also also, this seems unrelated to ADM so probably shouldn't be grouped with it.

Reply:
I just used the latest version suggested from IDE. Should I add the link to where it is in maven?

Reply:
Or to the github file line where it is declared?

Reply:
Preferably, if app-services has documentation with latest version, point to that. Otherwise, maybe point to the github file (I'd say maven first but it should be hosted on maven.mozilla.org which isn't really a full web server).